SEED-platform/seed
# Meter display settings updates

#### Any background context you want to provide?
See linked issue. Also, it was decided that a more appropriate default meter display setting for electric energy types is kWh (thousand Watt-hours).

#### What's this PR do?
In inventory detail meter views, meter unit display settings are found by starting with the PropertyView. The "currently selected" organization is no longer used to provide meter unit display settings.

A migration resets organizations' default meter settings. Among these settings, energy types are defaulted to `kWh (thousand Watt-hours)`. Note, since no actual users are using meter features yet, I don't anticipate that this reset would be an issue.

#### How should this be manually tested?
For any existing organizations, check that all the meter display settings are reset.

Upload meters to a property. For that property's org, change the display settings for the uploaded meter. View the property detail energy page. Switch between organizations, ideally switch to a few that have different display settings for the given meter type. Be sure that the display settings always use the settings of that property's org.

#### What are the relevant tickets?
#1920

#### Screenshots (if appropriate)
